### ISSUE TITLE
Android crash on new games with the graphical overmap mod enabled

### ISSUE THREAD

**Issue by Qrox** [CleverRaven/Cataclysm-DDA#36092]
# Describe the bug
Game crashes on new games with the graphical overmap mod enabled on the Android version.

# Steps To Reproduce
1. Create a world with the graphical overmap mode enabled on the Android version.
2. Choose "play now" to start a new game.
3. It crashes when loading the graphical overmap mod.

# Expected behavior
No crash whatsoever.

# Versions and configuration
- OS: Android
    - OS Version: Manufacturer: HUAWEI; Model: EVA-AL10; Release: 8.0.0; Incremental: 540(C00); 
- Game Version: 0.D-10157-gcf1dc76 [64-bit]
- Graphics Version: Tiles

# Additional context
no crash.log was generated

debug.log:
```
16:39:27.820 : Starting log.
16:39:27.820 INFO : Cataclysm DDA version 0.D-10157-gcf1dc76
16:39:29.365 INFO : SDL version used during compile is 2.0.11
16:39:29.365 INFO : SDL version used during linking and in runtime is 2.0.11
16:39:29.389 INFO : USE_COLOR_MODULATED_TEXTURES is set to 1
16:39:29.389 INFO : generate_alt_rect_texture() = FAIL. alt_rect_tex_enabled = 1
```

**Comment by JoeFreeman-Adv** [comment]
Possibly related to Issue [#35939](https://github.com/CleverRaven/Cataclysm-DDA/issues/35939)?

User Fay wrote on discord: ["I'm on android atm when I use set it up with how read me says the map is all white then if I try to reload application it crashes"](https://discordapp.com/channels/598523535169945603/642349261320880128/652738883863183362)

**Comment by ZhilkinSerg** [comment]
> 
> 
> Possibly related to Issue [#35939](https://github.com/CleverRaven/Cataclysm-DDA/issues/35939)?
> 

Nope. User just should make some steps from Readme and remove specific json file from mod folder.

**Comment by ZhilkinSerg** [comment]
Fixed in #35792
